src/components/organisms/tables/SelectVendorsTable/SelectVendorsTable.tsx

```diff
@@ -88,14 +88,19 @@ const SelectVendorsTable = <TFormValues extends FieldValues>({
   handleSortingChange,
   handlePaginationChange,
```

Comment:
[blocker]

filtering and sorting not working.
Skills are missing, but filtering works.

Reply:
[blocker]

In 'FreeDaysHearder' if we don't have info text then info icon should be hidden

Reply:
FreeDaysHearder proper information added, thx to Birgit.

Filtering on languages now works. Skills are added. When you say sorting not working, what do you mean? Which column does not work?